### DISCUSSION TITLE
Loginovo ZG101-ZL Operation Mode - 3 clics

### DISCUSSION THREAD

**Discussion by dsduarte** [Koenkk/zigbee2mqtt#22630] [Devices]
Hi all...
I have a few of those action butons accross the house and I have some automations based on button press events...
It supports single, double and hold but when it is 3 clics it changes the operation mode from event to command....
Is there a way to prevent this change to happen?
Some time the childreen mess them around and when a button press don't work as expected I never know if it's a network issue or if the operation mode has changed accidentally...
Thanks!

**Comment by davidperske** [comment]
I am using Z2M with home assistant. 

Created a automation that is triggered by the button changing mode from event to command, then the action posts a MQTT message that changes it back to event mode as described in the operation mode section of the device page https://www.zigbee2mqtt.io/devices/ZG-101ZL.html#loginovo-zg-101zl

Sometimes it seems to take an extra single click for this to complete and register in Z2M but appears to generally do what I need.

Bit of a workaround but enough for me to keep using these for a while if its stable

**Reply by dsduarte** [reply]
great idea!
I just did that on HA and its woring.
